src/functions/marshal_headers_from_sapi.php

```diff
@@ -35,7 +35,7 @@ function marshalHeadersFromSapi(array $server) : array
             }
         }
 
-        if ($value && strpos($key, 'HTTP_') === 0) {
+        if (($value || $value === "0") && strpos($key, 'HTTP_') === 0) {
```

Comment:
What about `$value !== ''` here?

Reply:
I think the same we should have in line 44, or maybe before:

```php
if ($value === '') {
    continue;
}
```

Reply:
I agree. Updated.